File: ProjectManager.Models/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ProjectManager.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }

        //Hány órát van alkalmazva hetente
        public int HoursPerWeek { get; set; }

        //Hány óra olyan dolga van ami nem a projektkhez kapcsolódik
        public int OtherThingsToDoForWeeks { get; set; }
        public List<EmployeeSkills> EmployeeSkills { get; set; }
        public List<ProjectForWeeks> ProjectsForWeeks { get; set; }
    }
}

File: ProjectManager.Web/ClientApp/src/viewmodels/employee.ts
import { Skill } from "./skill";
import { ProjectForWeeks } from "./projectforweeks";
import { SkillToProjectAndEmployee } from "./skillToProjectAndEmployee";

export class Employee {
  id: number;
  name: string;
  email: string;
  skills: Skill[];
  otherThingsToDoForWeeks: number;
  hoursPerWeek: number;
  employeeSkills: SkillToProjectAndEmployee[];
  projectsForWeeks: ProjectForWeeks[];
}

Employee.prototype.toString = function employeeToString () {
  return this.name;
}
